docs/plugins.rst

```diff
@@ -64,7 +72,14 @@ Once your config class is defined, you associate it with the plugin via the ``co
 Storage / Persisting Data
 --------------------------
 
-BasePlugin provides a wrapper around the Redis API that Plugins should use for storage. Since multiple channels can be using the same plugin, keys need to unique per plugin instance. BasePlugin takes care of this for you.
+BasePlugin provides a simple wrapper around the Redis API that Plugins should use for storage - it'll handle formatting each key so that it's unique to that Plugin.
+
```

Comment:
Since I know you've got a working BrainzBot setup I'm gonna ask for something a bit more advanced here: Can you figure out the format that is used to save values in redis and add a short explanation here? This may require fiddling with redis-cli a bit.

Reply:
Done.